.eslintignore

```diff
@@ -9,6 +9,6 @@ lib
 logs/
 node_modules/
 wptreport*.json
-wpt/
+./wpt/
```

Comment:
This is more correct as `tools/wpt` which I was thinking of moving things was ignored because of it.